src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="info">
+	<Properties>
+		<Property name="baseDir">logs</Property>
+		<Property name="logPattern">%d{YYYY-MM-dd HH:mm:ss.SSS} [%t] %-5level %l - %msg%n</Property>
+	</Properties>
+	<Appenders>
+		<Console name="Console" target="SYSTEM_OUT">
+			<PatternLayout pattern="${logPattern}" />
+		</Console>
+		<RollingFile name="mainLogFile"
+					 fileName="${baseDir}/Main-${date:MM-dd-yyyy_HH.mm.ss}.log"
+					 filePattern="${baseDir}/Main-%d{MM-dd-yyyy_HH.mm.ss}.log">
```

Comment:
Since this project is a template, maybe do something to indicate that "Main-" should be replaced with the project's name, such as `download-directory`?

Reply:
Sounds good - I can introduce a variable for the project name that can be changed based on the project.

I think I should also create a README of instructions on what files have placeholders that are expected to be changed.